src/main/java/edu/itu/cavabunga/controller/wrapper/ComponentPropertyResponse.java

```diff
@@ -0,0 +1,42 @@
+package edu.itu.cavabunga.controller.wrapper;
```

Comment:
we can consider package name wrapper to response.

Reply:
Fixed in 95800e3da98b799aafd20ecabdebd06e613b3473

Reply:
nope same, also response class should be part of core I guess

Reply:
sorry fixed in ca401aad7a8f592ca854ced692243564cc7ca377

Reply:
And we talked about using the single class for API http response. That is the issue if we use the same class with same object type for every response:

localhost:8080/participant/testuser/calendar
While using ComponentReponse which has "Component data" field, which contains Component's 

    "status": 0,
	"message": null,
	"data": [
		{
			"@type": "Calendar",
			"id": 1,
			"components": [
				{
					"@type": "Event",
					"id": 2,

localhost:8080/participant/testuser/calendar
While using GeneralResponse which has "Object data" field, which may contains any kind of object

    "status": 0,
	"message": null,
	"data": [
		[
			{
				"id": 1,
				"components": [
					{
						"@type": "Event",
						"id": 2,

"@type": "Calendar" => is dissapear in the second situation which is determines objec type of root object.